augur/parse.py

```diff
@@ -127,7 +127,7 @@ def parse_sequence(sequence, fields, strain_key="strain", separator="|", prettif
                                             etal='lower' if field.startswith('author') else None)
 
     # parse dates and convert to a canonical format
-    if fix_dates and 'date' in metadata:
+    if 'date' in metadata:
```

Comment:
Good catch! Fixed: bf4ff71dc4101b77d94eb2d968b0721d44e088e2

Reply:
Added context in a commit message: 95dccb771b19d8128fed04c699edd082f96835a5

I think this can count as a bug fix, though I'm only speculating and haven't tested implications of the bug.

Reply:
Thanks for adding more context! The commit message sounds right. 

(_non-blocking_) We could add another Cram test for `augur parse` without the `--fix-dates` flag.

Reply:
Done: 18768865b2d8743bf053ef4d1d612f5f8557dc0c...6b7a72cd87be01246dbee416ed708fd37ff924ea